scripts/qa_crowbarsetup.sh

```diff
@@ -5070,11 +5071,20 @@ function oncontroller_testpreupgrade
             complain 11 "No tempest image found. Please make sure the tempest barclamp created its images."
         fi
     fi
-    openstack stack create upgrade_test -t \
-        /root/scripts/heat/2-instances-cinder.yaml \
-        --parameter image=$tempest_image $heat_stack_params
-    wait_for 15 20 "openstack stack list | grep upgrade_test | grep CREATE_COMPLETE" \
-             "heat stack for upgrade tests to complete"
+    echo "starting $upgrade_test_stack_count test stack(s)..."
+    suffix=''
+    for i in $(seq 1 $upgrade_test_stack_count); do
+        openstack stack create upgrade_test$suffix -t \
```

Comment:
Uhm... some quoting here?

Reply:
hmmm... if you mean around `upgrade_test$suffix` then I don't think it's needed. `suffix` is not user provided so it should never contain spaces.

Reply:
Right, but you could say it about many other places that are safe when you are writing them until someones changes the conditions not knowing about all the effects...

Why not do it cleanly?

Reply:
done.
PS. same goes the other way: there are many places which don't have proper quoting already :o)